dist/angular-localization.js

```diff
@@ -21,7 +21,9 @@ angular.module('ngLocalize.InstalledLanguages', [])
     });
 angular.module('ngLocalize')
     .service('locale', ['$injector', '$http', '$q', '$log', '$rootScope', '$window', 'localeConf', 'localeEvents', 'localeSupported', 'localeFallbacks', function ($injector, $http, $q, $log, $rootScope, $window, localeConf, localeEvents, localeSupported, localeFallbacks) {
-        var TOKEN_REGEX = new RegExp('^[\\w\\.-]+\\.[\\w\\s\\.-]+\\w(:.*)?$'),
+        var TOKEN_REGEX = localeConf.validTokens ?
+                new RegExp(localeConf.validTokens) :
```

Comment:
since you have `localeConf.validTokens` set by default you can just use it here, no need to have a ternary and wrap in `new RegExp`